src/olympia/reviewers/templates/reviewers/emails/force_disable_addon.ltxt

```diff
@@ -0,0 +1,13 @@
+{% extends "reviewers/emails/base.ltxt" %}{% block content %}
+Due to issues discovered during the review process, your add-on {{ name }} has been disabled on addons.mozilla.org and no longer appears. Users who have previously installed your add-on will be able to continue using it.
+
+Please see the reviewer's comments below for more information.
+
+********
+Details:
+{{ comments }}
+********
+
+Thank you for your attention.
+
+{% endblock %}
```

Comment:
@wagnerand is this email copy okay?  I took one of the existing templates for rejections and slightly modified it.

Reply:
I forwarded it to Ashley and Lindsay

Reply:
From the slack thread it seemed like no changes were needed so resolving